libjlm/Makefile.sub

```diff
@@ -105,7 +105,7 @@ namespace jlm {\n \
   static inline filepath llcpath(\"$(shell $(LLVMCONFIG) --bindir)/llc\");\n \
   static inline filepath firtoolpath(\"$(FIRTOOL)\");\n \
   static inline filepath verilatorpath(\"$(VERILATOR_BIN)\");\n \
-  static inline filepath verilatorrootpath(\"$(VERILATOR_ROOT)\");\n \
+  static inline filepath verilatorrootpath(\"$(VERILATOR_PATH)\");\n \
```

Comment:
Are you sure that this is correct? We already have verilatorpath that points towards VERILATOR_BIN and then we also have verilatorrootpath that you changed from VERILATOR_ROOT to VERILATOR_PATH?

Reply:
Yes, it is currently how things work.

VERILATOR_PATH is the root directory of Verilator, since it compiles into a binary it needs to know where headers etc. are located. VERILATOR_BIN is the specific binary used to execute Verilator.

Reply:
From libjlm/Makefile.sub

# Default verilator for Ubuntu 22.04
VERILATOR_BIN ?= verilator_bin
VERILATOR_PATH ?= /usr/share/verilator